### PR TITLE
description text area is added above amount $ entry area, and is optional field

### PR DESCRIPTION
**Pull request by sonu-sonam**
added in both crypto bank transfer and payId

### CODE REVIEW

**Comment by radek1st** [comment]
and a cucumber test please :) even though it will fail before the backend is ready

**Comment by sonu-sonam** [comment]
> and a cucumber test please :) even though it will fail before the backend is ready

no, I think it will pass, I checked by transferring $0.01  it succeeded

**Comment by radek1st** [comment]
ah, but the description is ignored at the backend. You should do a get on the transaction id and verify if the parameters you sent match.

**Comment by sonu-sonam** [comment]
> ah, but the description is ignored at the backend. You should do a get on the transaction id and verify if the parameters you sent match.

ok, I will leave this until backend is ready then, is that alright?

**Comment by radek1st** [comment]
You can still do it, just you will not get the description from the backend

**Comment by radek1st** [comment]
@sonu-sonam backend has been updates, please add the cucumber test

File: tests/javascript-cucumber/Features/transaction.feature [hunk 1]
```diff
@@ -22,4 +21,15 @@ Scenario Outline: Transfer money less than $50 or more than $30,000 to bank acco
...
   |31000|
 
+  Scenario Outline: Create Transaction successfully
+    Given user navigates to Relaypay login page
+    And the user with "<username>" "<password>" successfully logs in
+    And user navigates to Relaypay bank transfer page
+    And fills in <bsb> <accountNumber> "<description>" <amount> for Transaction
+    When transfer button is clicked
+    Then user is redirected to payment page
+    Examples:
```
Comment (radek1st) [inline tests/javascript-cucumber/Features/transaction.feature:31]: And account activity page has transaction matching <bsb> <accountNumber> "<description>" <amount>

**Pull request merged**